### PR TITLE
docs: Update Linux instructions #554 [skip ci]

### PR DESCRIPTION
**Pull request by flying-sausages**
Moves install docs for native linux installs to their own chapter and divide them by tabs

### CODE REVIEW

File: docs/getting-started/installation.md [hunk 1]
```diff
@@ -57,38 +57,58 @@ docker run -d ...
...
-cd /opt/overseerr
+cd ~/.overseerr
 git pull
 yarn build
 yarn install
 yarn start
 ```
+{% endtab %}
+
+{% tab title="Arch" %}
```
Comment (flying-sausages) [inline docs/getting-started/installation.md:98]:
```suggestion
{% tab title="ArchLinux" %}
```

File: docs/getting-started/installation.md [hunk 2]
```diff
@@ -117,10 +137,4 @@ Docker on Windows works differently than it does on Linux; it uses a VM to run a
 
 ## ArchLinux \(Third party\)
```
Comment (flying-sausages) [inline docs/getting-started/installation.md:138]:
```suggestion
```

**Review by flying-sausages — COMMENTED** [review]
Arch references

File: docs/getting-started/installation.md [hunk 3]
```diff
@@ -57,38 +57,58 @@ docker run -d ...
...
-cd /opt/overseerr
+cd ~/.overseerr
 git pull
 yarn build
 yarn install
 yarn start
 ```
+{% endtab %}
+
+{% tab title="ArchLinux" %}
```
Comment (flying-sausages) [inline docs/getting-started/installation.md:98]:
```suggestion
{% tab title="ArchLinux \(3rd Party\)" %}
```

**Review by samwiseg0 — APPROVED** [review]
Thanks!

**Pull request merged**